source/features/unwrap-unnecessary-dropdowns.tsx

```diff
@@ -36,12 +37,45 @@ async function unwrapNotifications(): Promise<void | false> {
 	button.textContent = `Group by ${button.textContent.toLowerCase()}`;
 }
 
+function replaceRerunDropdown(menu: Element): void {
+	const dropdown = $('anchored-position', menu);
+	if (!dropdown) {
```

Comment:
Too many if else here. In what case we have an action-menu but not anchored-position?